nixos/modules/security/pam.nix

```diff
@@ -461,6 +473,7 @@ let
             account [success=ok default=ignore] ${pkgs.google-guest-oslogin}/lib/security/pam_oslogin_admin.so
           '' +
           ''
+            account required pam_unix.so
```

Comment:
What was the reason behind that btw? Haven't checked, but if it has to do something with ordering in the config file it should be documented in a comment here.

Reply:
The long description for this is [here](https://github.com/NixOS/nixpkgs/pull/183717#issuecomment-1200510251). But I will add a comment to this.

Reply:
Done.